tools/run_tests/xds_k8s_test_driver/README.md

```diff
@@ -125,22 +124,199 @@ python3 -m tests.baseline_test \
   --server_xds_port="$(($RANDOM%1000 + 34567))"
 ```
 
+## Local development
+This test driver allows running tests locally against remote GKE clusters, right
+from your dev environment. You need:
+
+1. Follow [installation](#installation) instructions
+2. Authenticated `gcloud`
+3. `kubectl` context (see [Configure GKE cluster access](#configure-gke-cluster-access))
+4. Run tests with `--debug_use_port_forwarding` argument. The test driver 
+   will automatically start and stop port forwarding using
+   `kubectl` subprocesses. (experimental)
+
 ### Setup test configuration
 
 There are many arguments to be passed into the test run. You can save the
-arguments to a config file for your development environment. Please take a look
-at
-https://github.com/grpc/grpc/blob/master/tools/run_tests/xds_k8s_test_driver/config/local-dev.cfg.example.
-You can create your own config by:
+arguments to a config file ("flagfile") for your development environment.
+Use [`config/local-dev.cfg.example`](https://github.com/grpc/grpc/blob/master/tools/run_tests/xds_k8s_test_driver/config/local-dev.cfg.example)
+as a starting point:
 
 ```shell
 cp config/local-dev.cfg.example config/local-dev.cfg
 ```
 
-### Clean-up resources
+Learn more about flagfiles in [abseil documentation](https://abseil.io/docs/python/guides/flags#a-note-about---flagfile).
+
+### Helper scripts
+You can use interop xds-k8s [`bin/`](https://github.com/grpc/grpc/tree/master/tools/run_tests/xds_k8s_test_driver/bin)
+scripts to configure TD, start k8s instances step-by-step, and keep them alive
+for as long as you need. 
+
+* To run helper scripts using local config:
+  * `python -m bin.script_name --flagfile=config/local-dev.cfg`
+  * `./run.sh bin/script_name.py` automatically appends the flagfile
+* Use `--help` to see script-specific argument
+* Use `--helpfull` to see all available argument
+
+#### Overview
+```shell
+# Helper tool to configure Traffic Director with different security options
+python -m bin.run_td_setup --help
+
+# Helper tools to run the test server, client (with or without security)
+python -m bin.run_test_server --help
+python -m bin.run_test_client --help
+
+# Helper tool to verify different security configurations via channelz
+python -m bin.run_channelz --help
+```
+
+#### `./run.sh` helper
+Use `./run.sh` to execute helper scripts and tests with `config/local-dev.cfg`.
```

Comment:
I'm open to that. What flags do you have in mind?

Reply:
Since this is an example, I'm gonna suggest all available flags:

- td_bootstrap_image
- network
- namespace
- firewall_source_range
- firewall_allowed_ports
- xds_server_uri
- server_name
- server_port
- server_maintenance_port
- server_xds_host
- server_xds_port
- client_name
- client_port
- force_cleanup
- check_local_certs

This is not a blocker for this PR.

Reply:
Ah. I was thinking about `local-dev.cfg.example` more as a minimal setup needed to hit the ground running. The idea is that you copy the file, change a couple of values, and it's ready to go. I want this to contain essentials + flags specific to local dev + flags likely to change.
I don't think it'll make sense to list all available flags in this file:

1. `--helpfull` already does it
2. Lower user entry barrier for the first setup
3. It's easy to get out of sync

Reply:
My end goal is hoping the user of this framework only need to look at one place to setup their environment quickly. The self-explanatory config design is common, like ssh configs, nginx configs. The un-important flags can be commented from start.

For the out-of-sync issue, this example can be generated from the `--helpfull` command.

Either way, I don't think this is urgent. PR LGTM.